modules/local/multiqc.nf

```diff
@@ -1,8 +1,8 @@
 process MULTIQC {
     label 'process_medium'
 
-    conda 'bioconda::multiqc=1.22.1'
-    container "community.wave.seqera.io/library/pip_multiqc:2c2e276ad8997cc4"
+    conda 'bioconda::multiqc=1.22.2'
```

Comment:
couldn't we use the modules implementation of multiqc?

Reply:
No because i have to pass specific inputs that the normal one does not take - same as in rnaseq [local multiqc](https://github.com/nf-core/rnaseq/blob/master/modules/local/multiqc/main.nf)